### PR TITLE
Changed asserts on commands output to json

### PR DESCRIPTION
**Pull request by sfc-gh-astus**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [x] I've added new automated tests to verify correctness of my new code.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [x] I've described my changes in the section below.

### Changes description
Changed asserts to check commands responses in JSON format, where it was possible

### CODE REVIEW

**Review by sfc-gh-jsikorski — DISMISSED** [review]

File: src/snowcli/cli/snowpark/procedure/commands.py [hunk 1]
```diff
@@ -36,11 +36,13 @@
...
 
 @app.command("init")
-def procedure_init() -> None:
+@with_output
+def procedure_init() -> OutputData:
     """
     Initialize this directory with a sample set of files to create a procedure.
     """
     create_project_template("default_procedure")
+    return OutputData()
```
Comment (sfc-gh-turbaszek) [inline src/snowcli/cli/snowpark/procedure/commands.py:45]: Let's return some message, "Project initialised" or something

File: src/snowcli/cli/snowpark/procedure/commands.py [hunk 2]
```diff
@@ -218,20 +221,23 @@ def procedure_update(
...
     package_native_libraries: str = PackageNativeLibrariesOption,
-):
+) -> OutputData:
     """Packages procedure code into zip file."""
     snowpark_package(
         pypi_download,  # type: ignore[arg-type]
         check_anaconda_for_pypi_deps,
         package_native_libraries,  # type: ignore[arg-type]
     )
+    return OutputData()
```
Comment (sfc-gh-turbaszek) [inline src/snowcli/cli/snowpark/procedure/commands.py:240]: How does it works? First we will do `print_db_cursor` and later empty output?

File: src/snowcli/cli/snowpark/procedure/commands.py (hunk 1 above)
Comment (sfc-gh-astus) [reply src/snowcli/cli/snowpark/procedure/commands.py:45]: For `OutputData()` we print `Done`. Is it ok?

File: src/snowcli/cli/snowpark/procedure/commands.py (hunk 2 above)
Comment (sfc-gh-astus) [reply src/snowcli/cli/snowpark/procedure/commands.py:240]: This command don't print cursor and `Output()` prints `Done`

Btw I removed `print_db_cursor`

**Review by sfc-gh-turbaszek — APPROVED** [review]
I'm ok with this change. However, the empt `OutpurData()` is bit ambiguous and we should really think about introducing some common messages.

**Pull request merged**